api-tester/src/main/java/com/revenuecat/apitester/java/PricingPhaseAPI.java

```diff
@@ -16,4 +16,10 @@ static void checkPricingPhase(PricingPhase pricingPhase) {
 
         OfferPaymentMode offerPaymentMode = pricingPhase.getOfferPaymentMode();
     }
+
+    static void checkPrice(Price price) {
+        String formatted = price.getFormatted();
+        Long amount = price.getAmountMicros();
+        String currencyCode = price.getCurrencyCode();
+    }
```

Comment:
While looking at other things, I noticed that the `Price` class didn't have API tests. This adds them.